### DISCUSSION TITLE
Connect first datapoint after initial spacing to the Y axis

### DISCUSSION THREAD

**Discussion by Simopich** [Abhinandan-Kushwaha/react-native-gifted-charts#250] [Q&A] [answered]
Title.
Is there a prop for this feature? Thanks in advance! :))

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @Simopich 👋
Thanks for requesting this feature and sorry for my late response.
This can be achieved using reference line. A reference line is a horizontal line starting at a given position from Y axis.

To connect the first data point to Y axis, we need to pass the prop `showReferenceLine1` and set the `referenceLine1Position` to the first data item value. And then pass width, thickness and type='solid' in the `referenceLine1Config` prop. The width should be equal to the value of `initialSpacing`. (Note that the default value of initialSpacing is 20).

Here is an example-
```js
const data=[
  {value: 90,},
  {value: 36},
  {value: 50},
  {value: 40},
  {value: 18,}
];
return <LineChart
data={data}
showReferenceLine1
referenceLine1Position={90}
referenceLine1Config={{
  type: 'solid',
  width: 20,
  thickness:2
}}
/>
```
The output of the above code is-

<img width="260" alt="Screenshot 2023-08-26 at 2 33 03 PM" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/20596944/3b11f6e7-73e9-4fcf-91e5-644e0454e99a">

I hope it helps🙂

**Reply by Simopich** [reply]
Thank you!
